PhysicsTools/PatAlgos/python/slimming/MicroEventContent_cff.py

```diff
@@ -171,3 +171,11 @@
 _phase2_timing_extraCommands = ["keep *_offlineSlimmedPrimaryVertices4D_*_*"]
 from Configuration.Eras.Modifier_phase2_timing_cff import phase2_timing
 phase2_timing.toModify(MicroEventContentMC, outputCommands = MicroEventContentMC.outputCommands + _phase2_timing_extraCommands)
+
+_pp_on_AA_extraCommands = [
+    'keep patPackedCandidates_hipixeltracks_*_*',
+]
+from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
+from Configuration.Eras.Modifier_pp_on_PbPb_run3_cff import pp_on_PbPb_run3
+(pp_on_AA_2018 | pp_on_PbPb_run3).toModify(MicroEventContent, outputCommands = MicroEventContent.outputCommands + _pp_on_AA_extraCommands)
+(pp_on_AA_2018 | pp_on_PbPb_run3).toModify(MicroEventContentMC, outputCommands = MicroEventContentMC.outputCommands + _pp_on_AA_extraCommands)
```

Comment:
I was included other changes because of renaming of modules. This comment is included in this commit : https://github.com/cms-sw/cmssw/pull/30313/commits/0c32060e85783640324132fec170d1405b7f9ff7